workers/redirector.js

```diff
@@ -12,17 +12,26 @@
 /**
  * `experimentPages` is an Array `[]` of Objects `{}` with the following key/value pairs:
  * - `targetPath` is the target pathname that the worker looks to match against (e.g. `/en-US/firefox/new/`).
+ * - `workerPath` is the pathname that cloudflare will send to this worker (e.g. `/en-us/firefox/*` .
```

Comment:
I feel like we're on the right path here, but adding this in `experimentPages` still feels like it might confuse people imho. Let's discuss later today 👍

Reply:
After discussion, moving to a different object, still in redirector.js.  It's a little bit more explicit.